### PR TITLE
Add scheduling content guide

### PR DESCRIPTION
**Pull request by gaapt**

### CODE REVIEW

File: docs/tutorials/general/scheduling-posts.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+page: :docsSchedulingPosts
+---
+
+#Overview
+
+In this article, we will expose how to manage the publishing time of content. 
+
+We use the example of a fictitious online magazine named `Andromeda` with `Articles` Content Type.  
+
+The editors of this magazine must manage content by specifying a publishing and unpublishing date and time for Entries.
```
Comment (grncdr) [inline docs/tutorials/general/scheduling-posts.md]: s/must/want to/

File: docs/tutorials/general/scheduling-posts.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Then, we order these matched Entries [by creation date](https://github.com/contentful-labs/scheduling_app/commit/d7fcab40cefcb1cc1f243dd3f50385c9b7e8c271):
+
+~~~ ruby
+f.cda_query = {content_type: "TE5C4G3m2AOwWcCoM6Cqc",'fields.endDateTime[gt]' => @timenow, 'fields.startDateTime[lte]' => @timenow, 'order' => 'sys.createdAt' }
+~~~
+
+## Adding a View
+
+ We [must add a view](https://github.com/contentful-labs/scheduling_app/commit/cc2243b34195808b7e5e5dedbd64ec9ea7adc284) in `sources/index.html.erb` to retrieve Entries:
```
Comment (grncdr) [inline docs/tutorials/general/scheduling-posts.md]: Let's emphasize here that we are doing this to make the app easier to use for our editors.

File: docs/tutorials/general/scheduling-posts.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Then, in `config.rb`, [we must define](https://github.com/contentful-labs/scheduling_app/commit/6bb6ad5a39149ed6cc9a772606072dabeee9f08c) a `timenow` ruby variable to filter scheduled Entries:
+
+~~~ ruby
+@timenow = Time.now - 900
+@timenow = @timenow.strftime("%FT%T")
+@timenow = @timenow.to_s
+~~~
+
+Note that it is good practice to subtract ~900 seconds(15 minutes) from the current time . It ensures that cached hits are also included in the response.
```
Comment (grncdr) [inline docs/tutorials/general/scheduling-posts.md]: This isn't quite accurate. What you want to do is _round_ to the nearest 15 (or even 5) minutes. The idea is that the URL is going to contain a query parameter like `endDateTime[gt]=2015-11-10T18:00:00` and you want that query parameter to remain stable over time.

To be honest now that this is a middleman app and not doing live requests, it's not such a big deal, but we should still show the appropriate best practice.

File: docs/tutorials/general/scheduling-posts.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+### Related resources
+
+ - For more advanced examples of using `contentful_middleman` check out [`contentful_middleman_examples`][cf-mm-examples] on GitHub.
+ - To learn more about the different kinds of filtering & querying supported by our API, check out [the reference documentation][filtering-reference].
+ - Finally, you can find more static site generator integrations on our [Tools][tools] page.
+
+[cf-mm-examples]: https://github.com/contentful-labs/contentful_middleman_examples
+[filtering-reference]: https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters
+[tools]: https://www.contentful.com/developers/docs/code/tools/
+
```
Comment (gaapt) [inline docs/tutorials/general/scheduling-posts.md]: Looks good to me.

**Comment by grncdr** [comment]
:+1:

**Pull request merged**